dialogflow_v2beta1/gapic/documents_client.py

```diff
@@ -353,24 +371,24 @@ def get_document(self,
         except AttributeError:
             pass
         else:
-            routing_metadata = google.api_core.gapic_v1.routing_header.to_grpc_metadata(
-                routing_header)
+            routing_metadata = google.api_core.gapic_v1.routing_header.to_grpc_metadata(routing_header)
             metadata.append(routing_metadata)
 
-        return self._inner_api_calls['get_document'](request,
-                                                     retry=retry,
-                                                     timeout=timeout,
-                                                     metadata=metadata)
-
-    def create_document(self,
-                        parent,
-                        document,
-                        retry=google.api_core.gapic_v1.method.DEFAULT,
-                        timeout=google.api_core.gapic_v1.method.DEFAULT,
-                        metadata=None):
+        return self._inner_api_calls['get_document'](request, retry=retry, timeout=timeout, metadata=metadata)
+
+    def create_document(
+            self,
+            parent,
+            document,
+            retry=google.api_core.gapic_v1.method.DEFAULT,
+            timeout=google.api_core.gapic_v1.method.DEFAULT,
+            metadata=None):
         """
         Creates a new document.
 
+        Note: The ``projects.agent.knowledgeBases.documents`` resource is
```

Comment:
Note that `projects.agent.knowledgeBases` resources are deprecated. Users should use `projects.knowledgeBases` (v2beta1)